CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessViewCategoryPanel.java

```diff
@@ -289,10 +323,72 @@ private void buttonLeftActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FI
         }
     }//GEN-LAST:event_buttonLeftActionPerformed
 
+    private void buttonUpActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_buttonUpActionPerformed
+        if (visibleList.getSelectedIndex() == -1) {
+            JOptionPane.showMessageDialog(OptionPanel, "No Category selected...", "Error", 1);
+        }
+        else
+        {
+            final int[] selectedIndices = visibleList.getSelectedIndices();
+
+            if(selectedIndices[0] != 0)
+            {
+                final int beforeIndex = selectedIndices[0]-1;
+                final List<String> selectedItems = visibleList.getSelectedValuesList();
+
+                visibleListModel.removeRange(selectedIndices[0], selectedIndices[selectedIndices.length - 1]);
```

Comment:
This code assumes the selected entries are in one contiguous block, when it's actually possible to have gaps in the selections. 
ie. Use Ctrl-Click to select the 2nd and 4th entries, leaving the 1st and 3rd entries unselected.
Then if you click the Up button, the 3rd entry will be removed, and not be added back in later code.

Probably best to not allow gaps in the selections.
Add these lines to the constructor:
```
        visibleList.setSelectionMode(ListSelectionModel.SINGLE_INTERVAL_SELECTION);
        hiddenList.setSelectionMode(ListSelectionModel.SINGLE_INTERVAL_SELECTION);
```